shub/cancel.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
+
+import click
+
+from scrapinghub import ScrapinghubAPIError
+from scrapinghub.client.utils import parse_job_key
+
+from shub.utils import get_scrapinghub_client_from_config
+from shub.config import get_target_conf
+from shub.exceptions import (
+    ShubException,
+    BadParameterException,
+    SubcommandException,
+)
+
+
+HELP = """
+Cancel multiple jobs from Scrapy Cloud.
+
+The cancel command expects the project ID (target) followed by
+the pair containing the spider ID and Job ID:
+
+    shub cancel 12345 1/1 1/2 1/3
+
+If the project ID is not defined it is going to use the default
+project (as defined in scrapinghub.yml):
+
+    shub cancel 1/1 1/2 1/3
+
+The cancel command requires a confirmation that could be skipped
+with the flag --force/-f:
+
+    shub cancel --force 1/1 1/2 1/3
+"""
+
+
+SHORT_HELP = "Cancel multiple jobs from Scrapy Cloud"
+
+
+@click.command(help=HELP, short_help=SHORT_HELP)
+@click.argument("target_or_key")
+@click.argument("keys", nargs=-1)
+@click.option('--force', '-f', is_flag=True)
+def cli(target_or_key, keys, force):
+    # Target may be defined or not
+    # When user doesn't specify then the target_or_key
+    # is just another key
```

Comment:
That's more a style-level nitpick, but I'd suggest to rephrase and shorten the comment, like:
```
# target_or_key contains a target or just another job key
```